src/go/flatgeobuf_test.go

```diff
@@ -90,7 +93,7 @@ func TestCreateFGBFileAndBasicSearch(t *testing.T) {
 		header := writer.NewHeader(headerBuilder).
 			SetName("Households ShapeFile Data").
 			SetTitle("Households ShapeFile Data").
-			SetGeometryType(flattypes.GeometryTypePolygon)
+			SetGeometryType(flattypes.GeometryTypeUnknown)
```

Comment:
If you move the changes to their own test, you do not need to use Unknown here (which again, goes for the "basic" part of this test). :)